Sources/NIO/EventLoop.swift

```diff
@@ -770,6 +770,14 @@ extension EventLoopGroup {
     }
 }
 
+/// Specifies how `EventLoopGroup` will be created and establishes lifecycle ownership.
+public enum EventLoopGroupProvider {
```

Comment:
This'll need to be renamed to `NIOEventLoopGroupProvider` as well (see: https://github.com/apple/swift-nio/blob/master/docs/public-api.md#1-no-global-namespace-additions-that-arent-prefixed)